### PR TITLE
Remove old copyright

### LINKED ISSUE #101: Correct copyright notices to reflect Copyright OpenSearch Contributors
Remove old copyright headers from all files. Similar to https://github.com/opensearch-project/anomaly-detection/issues/161

### PR DESCRIPTION
**Pull request by amitgalitz**
Signed-of-by: Amit Galitzky
### Description

 * Removed old copyright that was leftover from ODFE
 * Reformatted some files that were never formatted through saving it on visual studio code with prettier extension enabled.

### Issues Resolved

#101 

### Check List

- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by kaituo — APPROVED** [review]

File: public/components/ContentPanel/index.scss [hunk 1]
```diff
@@ -1,18 +1,3 @@
-/*
```
Comment (ylwu-amzn) [inline public/components/ContentPanel/index.scss:1]: Add new copyright header after removing this old one?

File: public/pages/AnomalyCharts/components/AlertsFlyout/alertsFlyout.scss [hunk 2]
```diff
@@ -1,27 +1,12 @@
-/*
```
Comment (ylwu-amzn) [inline public/pages/AnomalyCharts/components/AlertsFlyout/alertsFlyout.scss:1]: Add new copyright header

File: public/pages/AnomalyCharts/index.scss [hunk 3]
```diff
@@ -1,18 +1,3 @@
-/*
```
Comment (ylwu-amzn) [inline public/pages/AnomalyCharts/index.scss:1]: Add new copyright header and check other files?

Comment (amitgalitz) [reply public/pages/AnomalyCharts/index.scss:1]: Got it will check other files, didn't realize some files didn't have the other headers

**Comment by ohltyler** [comment]
@amitgalitz regarding reformatting changes - were you able to find a VS Code extension that is equivalent to running `prettier` from cli?

File: public/pages/AnomalyCharts/index.scss (hunk 3 above)
Comment (amitgalitz) [reply public/pages/AnomalyCharts/index.scss:1]: @ylwu-amzn @ohltyler Should I add a copyright header to files like `package.json` and `.prettierrc` also? A few more config files never had at any point a copyright header

**Comment by amitgalitz** [comment]
> @amitgalitz regarding reformatting changes - were you able to find a VS Code extension that is equivalent to running `prettier` from cli?

I still have not been able to find why the extension is slightly off from the cli or a different extension where I can guarantee output is 100% the same.

File: public/pages/AnomalyCharts/index.scss (hunk 3 above)
Comment (ohltyler) [reply public/pages/AnomalyCharts/index.scss:1]: If possible, sure. I remember there was some issue adding comments to some of those files, but if it compiles and works, go ahead.

File: public/components/ContentPanel/index.scss (hunk 1 above)
Comment (amitgalitz) [reply public/components/ContentPanel/index.scss:1]: added

File: public/pages/AnomalyCharts/index.scss (hunk 3 above)
Comment (amitgalitz) [reply public/pages/AnomalyCharts/index.scss:1]: You are right, can't add comments to json files and some others

**Comment by amitgalitz** [comment]
Reviewed each file that was changed to make sure nothing else was deleted except copyright and that each file that was changed has the new header

**Review by ohltyler — APPROVED** [review]

**Review by ylwu-amzn — APPROVED** [review]

**Pull request merged**